ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -200,6 +244,35 @@ device_classes: !ZenPackSpec
                                 dpName: volume_free
                                 lineType: AREA
                                 rpn: "size,-,size,/,-100,*"
+                    Physical Volume IO Stats:
+                        units: per sec
```

Comment:
"per sec" is not a unit. Units would be "merged/sec", "operations/sec", "sectors/sec" with "Read" and "Write" being graphpoint legends for each.